### PR TITLE
error with onnx2torch detail

### PR DESCRIPTION
**Pull request by zemm16**
```
onnx_model_path = "../data/models/birdnet-onnx-v1.onnx"
onnx_model = onnx.load(onnx_model_path)
torch_model_2 = convert(onnx_model_path)
```

gives the following error: "KeyError: 'unknown_242:0'"

Dug into onnx2torch's converter.py and onnx_model.graph and reproduced in following notebook:

emm-20230226-00-onnx2pytorch.ipynb

### CODE REVIEW

**Comment by acmiyaguchi** [comment]
Bleh, that's a bummer. What about trying the specific onnx version conversion to rule that out, since I did convert with opset 13 e.g.

```python
import onnx
from onnx import version_converter
import torch
from onnx2torch import convert

# Load the ONNX model.
model = onnx.load('model.onnx')
# Convert the model to the target version.
target_version = 13
converted_model = version_converter.convert_version(model, target_version)
# Convert to torch.
torch_model = convert(converted_model)
torch.save(torch_model, 'model.pt')
```
(from the README: https://github.com/ENOT-AutoDL/onnx2torch)

I dont think this is the issue, from your notebook:

> Below we see the output and input nodes, but no nodes specified in between.

I did have success running data through the entire network using the optimized graph and the onnx model, with the same outputs.

File: notebooks/emm-20230226-00-onnx2pytorch.ipynb [hunk 1]
```diff
@@ -0,0 +1,524 @@
+{
```
Comment (acmiyaguchi) [inline notebooks/emm-20230226-00-onnx2pytorch.ipynb:1]: <p>The other thing to try is to run the onnx conversion again, but this time with opset 17 and a single output corresponding to the embedding layer (242). You'll need to define a onnx_model_v2 here. </p><p> </p><pre class="ql-syntax" spellcheck="false">! python -m tf2onnx.convert \
  --opset 17 \
  --input {opt_graph_v1} \
  --inputs inputs:0 \
  --outputs unknown_242:0 \
  --output {onnx_model_v2}

</pre>
<br/>

 _Reply via <a href='https://app.reviewnb.com/dsgt-birdclef/birdclef-2023/pull/52/discussion/#comment-1118213935'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>

**Comment by acmiyaguchi** [comment]
Looks like this might be the end of the line -- are you okay with merging this as-is @zemm16 or is there any work that you want to append here?

**Comment by zemm16** [comment]
@acmiyaguchi no more work to upload on my end.

**Review by acmiyaguchi — APPROVED** [review]

**Pull request merged**